src/icu_info.cr

```diff
@@ -14,7 +14,7 @@ SOFILE   = "libicuuc.so"
 def icu_version : String
   version = nil
   if system("command -v icuinfo > /dev/null")
-    icuinfo = `icuinfo -v`
+    icuinfo = `icuinfo -v 2>/dev/null`
```

Comment:
Is there a specific reason to do it ? (I'd prefer to display errors at compile time, it can help to debug)
If not, can you remove it so I'll be able to merge the PR ?